src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/AdminAppealWithdrawnNotificationsEventsHandler.java

```diff
@@ -1,39 +0,0 @@
-package uk.gov.hmcts.reform.sscs.callback.handlers;
```

Comment:
Is removing of this event handler part of requirement?

Reply:
It was an exact replica what is done at FilterNotificationsEventsHandler. Don't want to duplicate the retry code.